packages/engineer/src/feature/dev-server.dev-server.env.ts

```diff
@@ -105,13 +124,21 @@ devServerFeature.setup(
                         port: actualPort,
                         inspect,
                         overrideConfig,
+                        externalFeatures,
                     },
                     resolvedSocketServerOptions
                 )
             );
 
             disposables.add(() => application.getNodeEnvManager()?.closeAll());
 
+            if (serveExternalFeaturesPath) {
+                serveStatic.push({
+                    route: `/${EXTERNAL_FEATURES_BASE_URI}`,
+                    directoryPath: externalFeaturesPath,
```

Comment:
this is the path of the base location of the external features (node_modules for example)